vignettes/rls.Rmd

```diff
@@ -223,33 +224,43 @@ rls_policies(con)
 #> 3 public     passwd    user_mod   PERMISSIVE {public}   UPDATE (CURR… ((CURRENT…
 ```
 
-## Grant permissions
+## Grant column level permissions
 
 Allow admin all normal rights
 
 
 ``` r
-dbExecute(con, "GRANT SELECT, INSERT, UPDATE, DELETE ON passwd TO admin")
+rls_tbl(con, "passwd") %>%
+  grant(select, insert, update, delete) %>%
+  to(admin) %>%
+  rls_run()
```

Comment:
Sorry to nit pick but I wonder if "run" is the right verb. My first instinct is "apply" but that's a very overloaded word in R. exec? perform? register?